UI/cmake/feature-macos-update.cmake

```diff
@@ -2,10 +2,6 @@ include_guard(DIRECTORY)
 
 find_package(nlohmann_json REQUIRED)
 
-if(NOT TARGET OBS::blake2)
-  add_subdirectory("${CMAKE_SOURCE_DIR}/deps/blake2" "${CMAKE_BINARY_DIR}/deps/blake2")
-endif()
-
```

Comment:
You'd probably have to ask @derrod for details.

Reply:
Nothing in `feature-macos-update.cmake` should require browser docks, but both WhatsNew and Sparkle rely on these components so if either is enabled this file needs to be included. The inclusion of blake2 was moved here because otherwise building OBS with Sparkle but without browser support would fail.

Reply:
Ah right, because we need connected the Sparkle functionality to our branch management. Should just be changed to an include guard then to avoid the same file being included twice (as it does right now).

Reply:
Thanks for clarifying and resolving.